### PR TITLE
Create 2022-11-02-monitor-airquality-airthings-netdata.md

### PR DESCRIPTION
**Pull request by shyamvalsan**
Monitoring indoor air quality with Airthings & Netdata

### CODE REVIEW

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+1. To start off you will need the API token and client id which can be created at https://dashboard.airthings.com/integrations/api-integration with the scopes ‘'’read:device:current_values’’’ checked.
+
+2. Export the secret key as an environment variable:
+
+```
+ export secret="secret-key"
+```
+
+3. Run this python script (after updating client_id and device_id variables from the Airthings website) as a cronjob piping the results to a csv file
```
Comment (andrewm4894) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:53]: Is there supposed to be a link or python script here

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+2. Export the secret key as an environment variable:
+
+```
+ export secret="secret-key"
+```
+
+3. Run this python script (after updating client_id and device_id variables from the Airthings website) as a cronjob piping the results to a csv file 
+
+4. We'll be using Netdata's new pandas collector to collect data from this CSV file (the same technique can be used to collect data from any CSV file and auto create powerful composite charts and dashboards).
```
Comment (andrewm4894) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:55]: Link to it

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
+---
+slug: airquality 
+title: "Monitor indoor air quality with Airthings and Netdata"
+description: "Monitor indoor air quality with Airthings and Netdata"
+tags: [airthings,airquality,indoor,netdata,co2,humidity,temperature,radon,pm1,pm25,voc,pressure]
+keywords: [airthings,airquality,indoor,netdata,co2,humidity,temperature,monitoring,how-to]
+authors: shyam
+---
+
+Monitoring indoor air quality with Airthings and Netdata.
```
Comment (andrewm4894) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:10]: Link to airthings

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md (hunk 1 above)
Comment (andrewm4894) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:53]: Instructions on how to set it as a cronjob maybe?

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+Each metric is presented on a separate time-series chart.
+
+![image](https://user-images.githubusercontent.com/24860547/199477936-d89f6601-ed62-4167-a7aa-239f6c44ceea.png)
+
+If you have multiple sensors, these charts will composite the information from all of the sensors with the option to filter the ones you want to explore further. 
+
+![image](https://user-images.githubusercontent.com/24860547/199477994-c4250231-a4aa-487e-a566-9c1a7b1466f2.png)
+
+Netdata comes with in-built unsupervised ML for anomaly detection, and each metric has an anomaly rate associated with it. This means that if the temperature or pressure or radon or any of these metrics starts behaving anomalously you can quickly spot it using Netdata.
```
Comment (andrewm4894) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:154]: Link to ml docs for those interested

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md (hunk 3 above)
Comment (shyamvalsan) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:10]:
```suggestion
Monitoring indoor air quality with [Airthings](https://www.airthings.com/) and Netdata.
```

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md (hunk 2 above)
Comment (shyamvalsan) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:55]:
```suggestion
4. We'll be using Netdata's new [pandas collector](https://learn.netdata.cloud/docs/agent/collectors/python.d.plugin/pandas) to collect data from this CSV file (the same technique can be used to collect data from any CSV file and auto create powerful composite charts and dashboards). 
```

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md (hunk 4 above)
Comment (shyamvalsan) [inline blog/2022-11-02-monitor-airquality-airthings-netdata.md:154]:
```suggestion
Netdata comes with in-built unsupervised ML for [anomaly detection](https://learn.netdata.cloud/guides/monitor/anomaly-detection), and each metric has an anomaly rate associated with it. This means that if the temperature or pressure or radon or any of these metrics starts behaving anomalously you can quickly spot it using Netdata.
```

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md (hunk 1 above)
Comment (shyamvalsan) [reply blog/2022-11-02-monitor-airquality-airthings-netdata.md:53]: will link it once the PR gets merged - could help merge it? https://github.com/netdata/community/pull/61

**Review by DShreve2 — APPROVED** [review]

File: blog/2022-11-02-monitor-airquality-airthings-netdata.md (hunk 1 above)
Comment (andrewm4894) [reply blog/2022-11-02-monitor-airquality-airthings-netdata.md]: done

**Pull request merged**